core/src/console/console_conf.cc

```diff
@@ -44,7 +44,7 @@ static bool SaveResource(int type, ResourceItem* items, int pass);
 static void FreeResource(BareosResource* sres, int type);
 static void DumpResource(int type,
                          BareosResource* reshdr,
-                         void sendit(void* sock, const char* fmt, ...),
+                         bool sendit(void* sock, const char* fmt, ...),
```

Comment:
PrintMessage is the only function called by this handler and this returns true, always.